packages/runtime/datastore-definitions/api-report/datastore-definitions.legacy.alpha.api.md

```diff
@@ -0,0 +1,153 @@
+## Alpha API Report File for "@fluidframework/datastore-definitions"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @beta @legacy
+export interface IChannel extends IFluidLoadable {
+    // (undocumented)
```

Comment:
Can we make sure this is documented now that we're "promoting" it to beta level?

Reply:
these aren't being promoted. this is the legacy alpha, so it just includes the existing beta